docs/extending/rich_text_internals.md

```diff
@@ -203,11 +203,11 @@ The editor interface used on rich text fields can be configured with the [WAGTAI
 
 It is possible to create your own rich text editor implementation. At minimum, a rich text editor is a Django **_class_ django.forms.Widget** subclass whose constructor accepts an `options` keyword argument (a dictionary of editor-specific configuration options sourced from the `OPTIONS` field in `WAGTAILADMIN_RICH_TEXT_EDITORS`), and which consumes and produces string data in the HTML-like format described above.
 
-Typically, a rich text widget also receives a `features` list, passed from either `RichTextField` / `RichTextBlock` or the `features` option in `WAGTAILADMIN_RICH_TEXT_EDITORS`, which defines the features available in that instance of the editor (see [rich text features](rich_text_features)). To opt in to supporting features, set the attribute `accepts_features = True` on your widget class; the widget constructor will then receive the feature list as a keyword argument `features`.
+Typically, a rich text widget also receives a `features` list, passed from either `RichTextField` / `RichTextBlock` or the `features` option in `WAGTAILADMIN_RICH_TEXT_EDITORS`, which defines the features available in that instance of the editor (see [rich text features](rich_text_features)). To opt into supporting features, set the attribute `accepts_features = True` on your widget class; the widget constructor will then receive the feature list as a keyword argument `features`.
```

Comment:
I think the original is correct. 

https://english.stackexchange.com/questions/422848/opt-into-vs-opt-in-to
https://forum.wordreference.com/threads/opt-in-or-opt-into.3560327/

Reply:
I also could be wrong here, need some grammar gurus.

Reply:
I agree - "opt in" works as a phrasal verb, and the "in" part shouldn't be merged with the following word (just as we would tend to say "you must log in to the system" rather than "you must log into the system").